rpc/backend/call_tx.go

```diff
@@ -372,12 +372,17 @@ func (b *Backend) DoCall(
 		return nil, errors.New("header not found")
 	}
 
+	var bzOverrides []byte
+	if overrides != nil {
```

Comment:
seems no difference as before, not sure why change to pointer

Reply:
make it optional

Reply:
Otherwise, it's a required argument in json-rpc

Reply:
you is right